android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListViewProxy.java

```diff
@@ -626,9 +626,10 @@ public void replaceSectionAt(int index, ListSectionProxy section,
 	 * @param itemIndex Index of item in section.
 	 */
 	@Kroll.method
-	public void scrollToItem(int sectionIndex, int itemIndex)
+	public void scrollToItem(int sectionIndex, int itemIndex, @Kroll.argument(optional = true) KrollDict animation)
 	{
 		final TiListView listView = getListView();
+		final boolean animated = animation != null && animation.optBoolean(TiC.PROPERTY_ANIMATED, true);
```

Comment:
It looks like `Ti.UI.ListView` used to support animated scroll. It just wasn't documented.
https://github.com/appcelerator/titanium_mobile/blob/9_2_X/android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListViewProxy.java#L159-L175
https://github.com/appcelerator/titanium_mobile/blob/9_2_X/android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/TiListView.java#L1063-L1079

From looking at the code, it would default to smooth scroll even if the options dictionary was not provided... which this code change loses. We should probably change the code to this...
```java
final boolean animated = (animation == null) || animation.optBoolean(TiC.PROPERTY_ANIMATED, true);
```